packages/applications-service-api/src/services/document.service.js

```diff
@@ -63,7 +108,17 @@ const getOrderedDocuments = async (caseRef, pageNo) => {
   return documents;
 };
 
+const getFilters = async (filter) => {
```

Comment:
Appreciate you wanted to keep this generic but we do need to exclude Stage 0  for the stage filter

Reply:
Done.